tests/HasTranslatableSlugTest.php

```diff
@@ -302,4 +303,75 @@ public function it_can_update_slug_with_non_unique_names_multiple()
             $this->assertSame($expectedSlug, $model->getTranslation('slug', 'en'));
         }
     }
+
+
+    /** @test */
+    public function it_can_resolve_route_binding()
+    {
+        $model = new TranslatableModel();
+
+        $model->setTranslation('name', 'en', 'Test value EN');
+        $model->setTranslation('name', 'nl', 'Test value NL');
+        $model->setTranslation('slug', 'en', 'updated-value-en');
+        $model->setTranslation('slug', 'nl', 'updated-value-nl');
+        $model->save();
+
+        // Test for en locale
+        $result = (new TranslatableModel())->resolveRouteBinding('updated-value-en', 'slug');
```

Comment:
Instead of calling these methods directly, you could register a route that uses model binding and check if you get the desired model. That would give more confidence that everything is working correctly in tandem with Laravel.

Routes can be registered from anywhere, so also in this very test.

Reply:
I did this in `#it_can_bind_route_model_implicit`